app/src/main/java/app/attestation/auditor/AttestationProtocol.java

```diff
@@ -896,16 +894,16 @@ private static Verified verifyStateless(final Certificate[] certificates,
 
         for (int i = 2; i < certificates.length; i++) {
             try {
-                new Attestation((X509Certificate) certificates[i]);
-            } catch (final Attestation.KeyDescriptionMissingException e) {
+                ParsedAttestationRecord.createParsedAttestationRecord(List.of((X509Certificate) certificates[i]));
+            } catch (final IOException | IllegalArgumentException e) {
```

Comment:
We added Attestation.KeyDescriptionMissingException to the previous library and it needs to be added to the new library to reliably detect whether the attestation extension is present.